src/SFA.DAS.EmployerIncentives/Interfaces/IEmailService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Threading.Tasks;
+using SFA.DAS.EmployerIncentives.InnerApi.Requests;
+
+namespace SFA.DAS.EmployerIncentives.Interfaces
+{
+    public interface IEmailService
+    {
+        Task SendBankDetailRequiredEmail(long accountId, SendBankDetailsEmailRequest sendBankDetailsEmailRequest);
```

Comment:
Could this just be a generic interface?

Task Send<T>(T email) 

?

Reply:
Done